### ISSUE TITLE
Test UI does not appear in activity bar

### ISSUE THREAD

**Issue by dschuessler** [hbenl/vscode-test-explorer#152]
I installed the Mocha Test Explorer, the Angular/Karma Test Explorer and with them the Test Explorer UI. You can see them installed in the screenshot. Yet the lab flask icon does not appear in the activity bar and the UI does not seem to be present or accessible.

![Bildschirmfoto 2020-07-17 um 22 21 17](https://user-images.githubusercontent.com/5774545/87828119-8fa9ef00-c87c-11ea-9230-7090a95a71cd.png)

VSCode 1.47
macOS 10.14.6

**Comment by hbenl** [comment]
Did you set `testExplorer.hideWhen` in your VSCode settings? Otherwise I don't know how this could happen.

**Comment by dschuessler** [comment]
Thanks for responding. I have this setting set to "never".

**Comment by hbenl** [comment]
> Thanks for responding. I have this setting set to "never".

Hmm, interesting. In this case it should always appear in the activity bar.
I still can't figure out how it is possible that it doesn't.

**Comment by dschuessler** [comment]
If there is any more information I can provide from my end, let me know.

**Comment by jounathaen** [comment]
Hi, I have the same problem with
- VSCodium
- Test Explorer UI 2.19.1
- Fedora 32 - Wayland

**Comment by jounathaen** [comment]
I found out that in my case the problem seems to be the VSCodium package repository. It seems to be different than the MS repository and for whatever reason the version from that repository does not work. Using the Microsoft Repository, the extension works again.

**Comment by hbenl** [comment]
> I found out that in my case the problem seems to be the VSCodium package repository

It turns out that the version of the extension published on open-vsx.org was broken. I republished it and now it should work.

**Comment by hbenl** [comment]
@dschuessler I just tried installing the Mocha and Karma Test Explorers in a fresh install of VSCode in MacOS 10.15 and it did work as expected (i.e. the Test Explorer UI appears and I could interact with Mocha tests). You could try the same (e.g. in the Guest user account of MacOS or by renaming `~/.vscode` and `~/Library/Application Support/Code`) to see if this issue is triggered by some other extension or some VSCode setting.

**Comment by dschuessler** [comment]
@hbenl This is not necessary. I had installed the extension via VSCodium as well and just checked that it is not a VSCodium related error by starting up VSCode using the same extension directory. So VSCode was using the version from Open VSX as well. It did not occur to me that the installation source could be important for the reproduction of the error.

I updated the extension and the icon is appearing now. Thank you for investigating this!
